### ISSUE TITLE
Accessibility Query

### ISSUE THREAD

**Issue by tomtdigital** [ntwcklng/gatsby-remark-embed-youtube#2]
Hi there. Appreciate this might not be straightforward, but is it please possible to add custom iframe title attributes for accessibility purposes? At present, we are failing e2e tests using the plugin in our md files. We were loving the plugin up until this point! Thanks for your time, Tom
